forem/forem
# Remove rack attack throttle for admin users in API requests

## What type of PR is this? (check all applicable)


## Description

Remove rack attack throttle for admin users in API requests

## Related Tickets & Documents

- Closes #18610

## QA Instructions, Screenshots, Recordings

Check tests are covering the use cases, otherwise attempt to make quick API calls on local environment using admin vs non-admin keys to test the Rack::Attack throttling.

### UI accessibility concerns?

None - backend change

## Added/updated tests?


## [Forem core team only] How will this change be communicated?


## [optional] What gif best describes this PR or how it makes you feel?

![terrible sonic drawing: gotta go fast](https://media.giphy.com/media/FzuiifgDBL3Lq/giphy.gif)
